Readme.md

```diff
@@ -253,3 +253,18 @@ template = local/foo.nomad
 ```
 
 Afterwards, run `./liquid deploy`, which will send your job `foo` to nomad.
+
+#### Enabling/disabling applications
+Applications can be enabled/disabled on deploy by setting them `on` or `off`
+in the `apps` section:
+```ini
+[apps]
+nextcloud = off
+```
+
+By default, all applications are started, but this default can also be changed
+in the `deploy` section:
+```ini
+[deploy]
+default_app_status = off
```

Comment:
If we're going to have this override, let's at least put it under `[apps]` right before all those other settings it switches.